packages/p2p/src/components/modal-manager/modals/market-rate-change-error-modal/market-rate-change-error-modal.tsx

```diff
@@ -2,20 +2,78 @@ import React from 'react';
 import { Button, Modal, Text } from '@deriv/components';
 import { localize, Localize } from 'Components/i18next';
 import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
-import { getTextSize } from 'Utils/responsive';
+import { useStores } from 'Stores';
+import './market-rate-change-error-modal.scss';
 
-const MarketRateChangeErrorModal = () => {
+type TMarketRateChangeErrorModal = {
+    submitForm: () => void;
+    values: {
+        currency: string;
+        input_amount: number;
+        local_currency: string;
+        received_amount: number;
+    };
+};
+
+const MarketRateChangeErrorModal = ({ submitForm, values }: TMarketRateChangeErrorModal) => {
     const { is_modal_open, hideModal } = useModalManagerContext();
+    const { buy_sell_store } = useStores();
+
+    React.useEffect(() => {
+        buy_sell_store.form_props.setIsMarketRateErrorModalOpen(true);
```

Comment:
what is this for? won't this open another modal?

Reply:
it won't open another modal, this is to track if the current modal is the market rate error modal, and I can't use the isCurrentModal(`MarketRateErrorModal`) as for some reason, the check doesn't return true despite it being displayed on responsive.... 

So i need to have the setter to check inside of the buy-sell-modal to not show the advertiser has changed the rate modal while the market rate modal is open on responsive